receiver/prometheusreceiver/internal/internal_test.go

```diff
@@ -64,14 +63,10 @@ func (m *mockConsumer) ConsumeMetricsData(ctx context.Context, md consumerdata.M
 	return nil
 }
 
-type mockMetadataSvc struct {
```

Comment:
Replaced mock of entire MetadataService with just a mock of ScrapeManager invoked by it, so `transaction_test.go` now catches a bug fixed in this PR.